src/lib/views/AlbumsView.svelte

```diff
@@ -65,31 +66,32 @@
 
     async function showCurrentlyPlayingAlbum() {
         if (!$currentSong) return;
+        
+        if (await updatePlayingAlbum()) {
+            currentAlbumElement?.scrollIntoView({
+                block: "center",
+                behavior: "instant"
+            });
+
+            isVisible = true;
+            isInit = false;
+        }
+    }
+    
+    async function updatePlayingAlbum() {
         // Strip the song from album path
-        const albumPath = $currentSong.path.replace(
-            `/${$currentSong.file}`,
-            ""
-        );
+        const albumPath = await path.dirname($currentSong.path)
         // Find the album currently playing
         currentAlbum = await db.albums.get(
             md5(`${albumPath} - ${$currentSong.album}`.toLowerCase())
         );
-        if (!currentAlbum) return;
-        // $albumPlaylist = tracks;
-        // $playlistIsAlbum = true;
-
-        // Scroll to album
+        if (!currentAlbum) return false;
+        
         currentAlbumElement = document.querySelector(
             `[data-album='${currentAlbum.id}']`
         );
-
-        currentAlbumElement?.scrollIntoView({
-            block: "center",
-            behavior: "instant"
-        });
-
-        isVisible = true;
-        isInit = false;
+        
```

Comment:
Let's also add a `onScroll()` call here to pop up the "Scroll to now playing" when scroll pos hasn't changed, but a new albums comes on and is out of view. Just make sure `container` is not null

Reply:
I've moved the `onScroll()` to the caller since the container might not be mounted when this function is called.